### PR TITLE
refactor: shadowing built ins

### PR DESCRIPTION
**Pull request by florian-hoenicke**

### CODE REVIEW

**Review by JoanFM — APPROVED** [review]

**Review by theUnkownName — DISMISSED** [review]

File: jina/docker/helper.py [hunk 1]
```diff
@@ -63,5 +63,5 @@ def sleep(self, seconds=5):
         self.logger.debug(f'sleeping for {seconds} seconds')
         time.sleep(seconds)
 
-    def __exit__(self, type, value, traceback):
```
Comment (hanxiao) [inline jina/docker/helper.py:66]: this is inherited from the exit stack builtin, i don't think we need to change that

Comment (hanxiao) [reply jina/docker/helper.py:66]: by inherited, i mean if you type `__exit__` in Pycharm, it auto-complete with this signature

**Review by hanxiao — CHANGES_REQUESTED** [review]
keep `__exit__` signature as `object.__exit__`, others look good

File: jina/docker/helper.py (hunk 1 above)
Comment (hanxiao) [reply jina/docker/helper.py:66]: okay my bad, somehow this one is not correctly auto-filled by Pycharm. The correct signature should be 
`object.__exit__(self, exc_type, exc_value, traceback)`

let's change to that
https://docs.python.org/3.7/reference/datamodel.html#object.__exit__

Comment (florian-hoenicke) [reply jina/docker/helper.py:66]: thanks, I changed it

**Review by theUnkownName — APPROVED** [review]

**Review by hanxiao — APPROVED** [review]

**Pull request merged**